### PR TITLE
[FSD-1373] Documentation for exposing store settings

### PR DESCRIPTION
**Pull request by acdameli**
[related](https://github.com/stagebloc/core/pull/5326)

### CODE REVIEW

File: api_v1.md [hunk 1]
```diff
@@ -1972,6 +1972,46 @@ This endpoint can be used to refund an entire order.
...
+	        "handling_fee_per_order": 0,
+	        "handling_fee_per_item": 0,
+	        "handling_fee_per_item_minimum_threshold": 0,
+	        "cart_lock_time_seconds": 0,
+	        "cart_require_billing_address": false,
+	        "cart_require_phone_number": false,
+	        "payment_preauth": false,
+	        "payment_verification": false
+	    }
+	}
```
Comment (stagebloc-iceman) [inline api_v1.md]: This should follow the same pattern set by other endpoints. There should be a definition section (w/out metadata wrapper) with explanations of anything necessary, then a Retrieve - Settings. It's a bit verbose for just one endpoint but it follows the pattern and makes it a little easier to redo this later for moving object definitions into a single list elsewhere.

File: api_v1.md [hunk 2]
```diff
@@ -1972,6 +1972,46 @@ This endpoint can be used to refund an entire order.
...
+
+This endpoint is used to get secure information about your store. It can only be accessed using an authentication token for an admin with store order creation access.
+
+	{
+	    "metadata": {
+	        "http_code": 200
+	    },
+	    "data": {
+	        "payworks_merchant_identifier": null,
+	        "payworks_merchant_secret_key": null
```
Comment (stagebloc-iceman) [inline api_v1.md]: We should include sample values for these, like `"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"` and `"abcdefghijklmnopqrstuvwxyz123456"` so people have some idea what the value contains and its length will be, or that information should be explained below here like in other areas (e.g. `Definition - Store Item`)

**Comment by acdameli** [comment]
@stagebloc-iceman missed unwrapping the full message there.

**Comment by acdameli** [comment]
release after: https://github.com/stagebloc/core/pull/5271

**Pull request merged**